File: src/main/java/MachTeacher/MachTeacher/dto/SosResponseDto.java
package MachTeacher.MachTeacher.dto;

public record SosResponseDto(
                Long id,
                SosUserDto student,
                SosUserDto acceptedBy,
                String subject,
                String message,
                String status,
                String createdAt,
                String acceptedAt,
                Long sessionId,
                Long conversationId) {
}

File: MachTeacher/app/src/main/java/com/example/machteacher/dto/SosResponseDto.kt
package com.example.machteacher.dto

data class SosUserDto(
    val id: Long,
    val fullName: String?
)


data class SosResponseDto(
    val id: Long,
    val student: SosUserDto?,
    val acceptedBy: SosUserDto?,
    val subject: String?,
    val message: String?,
    val status: String,
    val createdAt: String?,
    val acceptedAt: String?,
    val sessionId: Long?,
    val conversationId: Long?
)
